lib/util.js

```diff
@@ -8,6 +8,8 @@ module.exports.labelMaker = function(str) {
 
 module.exports.tokens = function(options) {
   return {
+    name: options.projectName,
+    nameFunc: options.projectName.replace(/-/, '_'),
```

Comment:
For consistency across generator efforts, we should probably keep this as projectName and machineName. Also `replace()` for a simple case like this will accept a string.